application/src/main/java/org/thingsboard/server/service/update/DeprecationService.java

```diff
@@ -63,4 +72,33 @@ private void checkQueueTypeDeprecation() {
                 )));
     }
 
+    private void checkDatabaseTypeDeprecation() {
+        String deprecatedDatabaseType = "timescale";
+        var fieldsWithDeprecatedType = new ArrayList<String>();
+
+        addFieldIfValueDeprecated(tsType, "ts", fieldsWithDeprecatedType, deprecatedDatabaseType);
+        addFieldIfValueDeprecated(tsLatestType, "ts_latest", fieldsWithDeprecatedType, deprecatedDatabaseType);
+
+        if (fieldsWithDeprecatedType.isEmpty()) {
+            return;
+        }
+
+        String deprecatedFieldNames = String.join(", ", fieldsWithDeprecatedType);
```

Comment:
Please simplify this. We don't need the "deprecatedFields". Just tell the user a generic "Timescale will no longer be supported as a storage provider" if ts or ts_latest type is Timescale